packages/lit-dev-content/src/components/litdev-search.ts

```diff
@@ -240,8 +257,30 @@ export class LitDevSearch extends LitElement {
 
   private _listboxClicked = false;
 
-  update(changed: PropertyValues<this>) {
-    if (!this._searchText && this._inputEl) {
+  private _lastSuggestions: Suggestion[] = [];
+
+  private get _currentSuggestions() {
+    if (!this._searchTask || this._searchTask.status !== TaskStatus.COMPLETE) {
+      return this._lastSuggestions;
+    }
+
+    this._lastSuggestions = this._searchTask.value!;
+    return this._searchTask.value!;
+  }
+
+  private get _isExpanded() {
+    return (
+      this._isFocused &&
+      this._currentSuggestions.length > 0 &&
+      !this._closedByEscape
+    );
+  }
+
+  update(changed: Map<string, unknown>) {
+    const textDoesntMatch = this._searchText !== this._inputEl.value;
+    const isSSRHydrate = this._inputEl && textDoesntMatch && !this.hasUpdated;
```

Comment:
Can you add a little more comment about this code? This is the first I'm hearing about SSR + hydration issues with inputs. Do we need to file an issue and track this?

Reply:
This is specifically because if you type in there and then it hydrates, because `@input` did not update `this._searchText` the `live()` directive will overwrite what is inside the `<input>` with `''` and the `live()` directive is necessary because IIRC safari on ios will make the cursor jump to the end if you set the value needlessly while editing it

Reply:
comment added LMK what you think

Reply:
ooh, can you file an SSR issue describing this? live() might not be the exact right thing for this case, but not sure yet.

Reply:
filed